### DISCUSSION TITLE
Move code to GitLab or Codeberg

### DISCUSSION THREAD

**Discussion by julianfairfax** [pine64/OpenPineBuds#87] [Ideas]
It would be nice if the project's code were moved to GitLab or Codeberg. GitHub is not an open source platform, and being owned by Microsoft makes it more of a danger to open source projects, rather than a help.

[Pine64 already has a presence on GitLab.](https://gitlab.com/pine64-org)

**Comment by shymega** [comment]
I want to stick with GitHub. I see no issue with it at this time, and we have a comprehensive CI setup with GitHub.

Microsoft is not that bad. I use Microsoft 365 with Linux machines no problem. I don't want this project to become a political statement.

**Reply by julianfairfax** [reply]
> I want to stick with GitHub. I see no issue with it at this time, and we have a comprehensive CI setup with GitHub.

That's alright, it's your decision. Feel free to close this then.

> Microsoft is not that bad. I use Microsoft 365 with Linux machines no problem. I don't want this project to become a political statement.

I understand that.

**Reply by shymega** [reply]
Thank you.

I've raised with Pine64 about syncing GitLab repos to GitHub, and GitHub repos to GitLab. This should allow people to use a FOSS platform with the project. The question remains of issues and MRs/PRs, but I can see that becoming messy.

I also find the GitLab UI very clunky to be honest - my browser lags on it.

**Comment by shymega** [comment]
Converting to a Discussion, as this isn't a bug.

**Comment by Ralim** [comment]
I dont really see any reason to move this now that we are already setup, we could setup a mirror to support access via Gitlab if this adds meaningful help to others.
We of course can move at any time without much drama should a pertinent issue show up.
(Github is also faster for me here which is nice, but its a super minor of a reason).

Can you provide any key reasons or advantages we would utilise in moving the repository?
